ruleset/rules/0803-attack-TA0003.xml

```diff
@@ -0,0 +1,21 @@
+<group name="TA0003,">
+    <rule id="92035" level="6">
+        <if_sid>60106</if_sid>
+        <field name="win.eventdata.ipAddress" type="pcre2">(?:[0-9]{1,3}\.){3}[0-9]{1,3}</field>
+        <field name="win.eventdata.ipAddress" type="pcre2" negate="yes">127\.0\.0\.1</field>
```

Comment:
Maybe you can use _localhost_ instead

Reply:
I got some samples on this (I mean local login events) and they seem to only use numeric format for ip address